apps/backend/src/libs/plugins/file-upload/file-upload.plugin.ts

```diff
@@ -30,13 +30,16 @@ const fileUpload = fastifyPlugin<Options>(
 			}
 
 			const data = await request.file();
+			const buffer = await data?.toBuffer();
 
-			if (
+			const isFileCorrect =
 				data &&
-				allowedExtensions.includes(data.mimetype as ValueOf<typeof ContentType>)
-			) {
-				const buffer = await data.toBuffer();
+				buffer &&
+				allowedExtensions.includes(
+					data.mimetype as ValueOf<typeof ContentType>,
+				);
 
+			if (isFileCorrect) {
```

Comment:
```suggestion
			if (!isFileCorrect) {
			  throw ...
```
To avoid nested if statements